freetextresponse/private/view.less

```diff
@@ -66,11 +74,17 @@
     display: inline-block;
   }
 
-  .word-count-error {
-    color: #A50000;
+  //  DEBUG - should have access to built-int
+  //  capa_alert in need in lms and studio
+  .capa_alert {
+    margin-top: 10px; 
+    padding: 8px 12px;
+    border: 1px solid #ebe8bf;
+    border-radius: 3px;
+    background: #fffcdd;
+    font-size: 0.9em;
   }
 
-  .submission-received {
-    color: #2EA41E;
-  }
+  // Found on internet
```

Comment:
old css thing I was using before switching to capa css classes.  Left in the comment.